src/Analyzers/VisualBasic/Tests/ValidateFormatString/ValidateFormatStringTests.vb

```diff
@@ -18,24 +19,18 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.ValidateFormatStri
             Return (New VisualBasicValidateFormatStringDiagnosticAnalyzer, Nothing)
         End Function
 
-        Private Function VBOptionOnCSharpOptionOff() As IDictionary(Of OptionKey2, Object)
-            Dim optionsSet = New Dictionary(Of OptionKey2, Object) From
-            {
-                {New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.CSharp), False},
-                {New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.VisualBasic), True}
-            }
-
-            Return optionsSet
+        Private Function VBOptionOnCSharpOptionOff() As IOptionsCollection
+            Return OptionsSet(
+                (New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.CSharp), False),
+                (New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.VisualBasic), True)
+            )
         End Function
 
-        Private Function VBOptionOffCSharpOptionOn() As IDictionary(Of OptionKey2, Object)
-            Dim optionsSet = New Dictionary(Of OptionKey2, Object) From
-            {
-                {New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.CSharp), True},
-                {New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.VisualBasic), False}
-            }
-
-            Return optionsSet
+        Private Function VBOptionOffCSharpOptionOn() As IOptionsCollection
+            Return OptionsSet(
+                (New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.CSharp), True),
+                (New OptionKey2(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatCalls, LanguageNames.VisualBasic), False)
+            )
```

Comment:
nit: we try to avoid naked close parens in IDE.

Reply:
Ah sure, I will fix it given I already have merge conflicts. Just curious - do we have any CodeStyle analyzer that will flag this?